ui/dashboards/src/components/Panel/PanelActions.tsx

```diff
@@ -68,9 +70,15 @@ export const PanelActions: React.FC<PanelActionsProps> = ({
   description,
   descriptionTooltipId,
   links,
-  queryResults,
+  panelContentProps,
```

Comment:
as we are only using the queryResults, we probably don't need to pass it wrapped into the panelContentProps